test/helpers.js

```diff
@@ -153,19 +153,20 @@ Runner.prototype.npmVersion = function(version){
 	if(arguments.length === 0) {
 		return this._version; 
 	}
-
-	this.algorithm = version >= 3 ? "flat": undefined;
 	this._version = version;
-	this._addVersion();
```

Comment:
you shouldn't set a NPM version by using `.npmVersion()`. 
the old `.npmVersion()`. overwrites the `npmAlgoritm` but not the `isFlatFileStructure`
the way we check the npm version is by scanning the npmAlgoritm in `package.system`.

so if you want to using npm < 3 you have to set a `nested` property in `package.system` like
https://github.com/stealjs/system-npm/pull/146/files#diff-e9a7e88b9a1c4fff3b21fb49ed79b8d7R104